packages/build-tools/src/builders/android.ts

```diff
@@ -100,7 +98,7 @@ async function buildAsync(ctx: BuildContext<Android.Job>): Promise<void> {
           expoApiServerURL,
           robotAccessToken,
           paths: cachePaths,
-          key: cacheKey,
+          key: CACHE_KEY_PREFIX + cacheKey,
```

Comment:
Hm, I think we may want to import `CACHE_KEY_PREFIX` from some constants file. This way it will be _the same_ for `eas/build` and the regular build process.

Maybe `generateDefaultBuildCacheKeyAsync` should accept `({ workingDirectory, platform })`?